src/components/dialogs/create-contingency-list-dialog.js

```diff
@@ -258,13 +341,21 @@ export const CreateContingencyListDialog = ({ open, onClose }) => {
                         disabled={
                             contingencyListName === '' ||
                             !contingencyNameValid ||
-                            loadingCheckContingencyName
+                            loadingCheckContingencyName ||
+                            (contingencyListType === ContingencyListType.FORM &&
```

Comment:
I know it's duplicated code, but maybe add this check in the beginning of `handleCreateNewContingencyList` as well ?

Reply:
Yes, done